modules/version/ComponentsVersions.js

```diff
@@ -63,23 +63,19 @@ function(node, mucResource, mucJid) {
 
         var version = item.value;
         if (this.versions[componentName] !== version) {
-            if(this.versions[componentName] &&
```

Comment:
I think the condition is wrong I can check that. But other than that if JitsiConnection._reload is triggered the client can end up on different server side deployment. So we need this code, I can add a comment there and change the name. WDYT?